modules/@shopify/checkout-sheet-kit/android/src/main/java/com/shopify/reactnative/checkoutsheetkit/CustomCheckoutEventProcessor.java

```diff
@@ -48,9 +55,20 @@ public void onCheckoutCompleted() {
     sendEvent(this.reactContext, "completed", null);
   }
 
+  @Override
+  public void onWebPixelEvent(@NotNull PixelEvent event) {
+    try {
+      ObjectMapper mapper = new ObjectMapper();
+      String data = mapper.writeValueAsString(event);
```

Comment:
completely untested, but wondering if we might be able to do something like 

```
Map<String, Object> map = objectMapper
      .convertValue(event, new TypeReference<Map<String, Object>>() {});
```

to use a similar approach to swift